CITATION.cff

```diff
@@ -1,20 +1,41 @@
 cff-version: 1.2.0
-message: "If you use this software, please cite it as below."
+message: |
+  Please cite this software using these metadata. "*" indicates co-equal authorship.
+
+title: "QGreenland"
+doi: "10.5281/zenodo.4558266"
```

Comment:
This DOI links to the v1.1 release. Is that expected? Should this be hard-coded here?

Reply:
:eyes:

Reply:
9f6a76fe Good catch!